go.mod

```diff
@@ -242,6 +261,14 @@ require (
 )
 
 //Note: add tag for go-ethereum before cutting hard-fork release
-replace github.com/ethereum/go-ethereum => github.com/iotexproject/go-ethereum v0.5.0
+replace github.com/ethereum/go-ethereum => github.com/iotexproject/go-ethereum v1.7.4-0.20250102060848-adaefd28e52f
```

Comment:
why we need this, and `go-pkgs` at L268?

Reply:
refer to previous comments https://github.com/iotexproject/iotex-core/pull/4551#discussion_r1952368005